packages/veto-frontend/src/components/molecules/SendBTC/SendBTC.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react'
+import { Input, Label, Text, Box, Select, Button, Flex } from '@veriphi/veto-ui'
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons/faChevronRight'
+import { faChevronLeft } from '@fortawesome/free-solid-svg-icons/faChevronLeft'
+import { useInput } from 'hooks/useInput'
+import { spend } from 'api'
+
+type Status = 'constructTx' | 'reviewTx' | 'sending' | 'successfulSend' | 'error'
+
+const SendBTC = () => {
+  // General State
+  const [status, setStatus] = useState<Status>('constructTx')
+  const [error, setError] = useState('')
+  const [successData, setSuccessData] = useState({ status: '', hash: '' })
+
+  // Input
+  const amount = useInput('')
+  const address = useInput('')
+  const fees = useInput('')
+  const label = useInput('')
+
+  // Handlers
+  const handleGoToReview = () => {
+    // just make sure we have the important bits to construct the transaction
+    if (!amount.value || !address.value || !fees.value) {
+      return
+    }
+    setStatus('reviewTx')
+  }
+
+  const handleSend = async () => {
+    setStatus('sending')
+    try {
+      const response = await spend({ address: address.value, amount: amount.value, eventMessage: label.value })
+      setSuccessData({ hash: response.hash, status: response.status })
+      setStatus('successfulSend')
+    } catch (e) {
```

Comment:
Its an unpopular opinion but I think all single letter variable name should be avoided (even i,j,k & e).
`i` is a bit of a stretch but can easily be replaced by `index` or `key` or `name` or anything else that represent the value.

`e` can be `error`, `exception`, `event`, `expect` or anything else that starts with an `e`
Without looking at the context/implementation is hard to know what the variable is used for.
Now in the case of `catch(e)` its a bit more obvious because the `catch` tells us its an `error` but it might not be obvious when you read the rest of the catch block.

It could be `setError(event.message)`, `setError(expect.message)`, `setError(environment.message)`, etc.

Reply:
Great case! I will adopt the standard :)